keps/sig-storage/3762-persistent-volume-last-phase-transition-time/README.md

```diff
@@ -618,32 +593,33 @@ feature.
 NOTE: Also set `disable-supported` to `true` or `false` in `kep.yaml`.
 -->
 
-Yes. This will result in the timestamp value being eventually set to `nil`. More details in
-"Upgrade / Downgrade Strategy" section.
+Yes, for PVs not updated while feature was enabled. However once the `LastPhaseTransitionTime` value is set, disabling
+feature gate will not remove the value.
+
+More details in "Upgrade / Downgrade Strategy" section.
 
 ###### What happens if we reenable the feature if it was previously rolled back?
 
-No issues expected. There are three cases that can occur for a PV:
+No issues expected. There are two cases that can occur for a PV:
 
   1. PV did not transition its phase when feature gate was enabled - the `LastPhaseTransitionTime` field was not added 
      to the PV object so this is the same case as enabling the feature gate for the first time.
-  2. PV did transition its phase but the `LastPhaseTransitionTime` *was not* reset to `nil` because the PV was not 
-     validated when the feature was enabled - the timestamp value will be updated on next phase change as if the feature
-     gate was never disabled.
-  2. PV did transition its phase but the `LastPhaseTransitionTime` *was* reset to `nil` because the PV was validated at
-     least once already while feature was enabled - the timestamp value will be updated on next phase change because
-     updates from `nil` are allowed.
+
+  2. PV did transition its phase when feature gate was enabled - the `LastPhaseTransitionTime` field is already set,
+     and it's timestamp value will be updated on next phase change.
 
 See "Upgrade / Downgrade Strategy" and "Notes/Constraints/Caveats" sections for more details.
 
 ###### Are there any tests for feature enablement/disablement?
 
-Unit tests for enabling and disabling feature gate will be added when transitioning to beta. See "Graduation criteria"
-section.
+Unit tests for enabling and disabling feature gate are required for alpha - see "Graduation criteria" section.
 
 The tests should focus on verifying correct handling of the new PV field in relation to feature gate state. Correct
-handling means the values of the newly added field are added or updated when PV transitions its phase and cleared when
-the feature is disabled.
+handling means the values of the newly added field are added or updated when PV transitions its phase while feature gate
+is enabled, and persisted if already set and feature gate is disabled.
+
+Feature enablement tests:
+https://github.com/RomanBednar/kubernetes/blob/294f5c9a42fead4a4cc75340a6b9171c9c657b3e/pkg/registry/core/persistentvolume/strategy_test.go#L45
```

Comment:
The link should lead to kubernetes/kubernetes, the commit is already merged there

Reply:
The link is wrong indeed - fixed. Thank you for review!